nucliadb_protos/utils.proto

```diff
@@ -104,3 +104,8 @@ message UserVectorSet {
 message UserVectorsList {
     repeated string vectors = 1;
 }
+
+enum VectorSimilarity {
+    Cosine = 0;
+    Dot = 1;
```

Comment:
To maintain a consistent style, use upper case in enums

Reply:
done